USE-CASES.md

```diff
@@ -180,6 +183,77 @@ Such questions are answerable if the defender has the requisite “capabilities,
 
 This section lists the capabilities that we have identified as useful for our key use cases. (Think of these as the requirements we have for our key use cases, deferring implementation opportunities or challenges to a subsequent design proposal)
 
+#### Recognize whether the same device is seen again in the context of the same identity
+This applies to both within the same domain or across domains. Detecting a same device returning would traditionally happen through storing IDs persistently in the browser and reusing them when a device is seen again. The lifespan of detection needs to be at a minimum 7 days.
+
+##### Use cases
+Account Creation: Seeing an identity being used with a device it has never been seen with before (or that's very different from the devices it has been seen with before) can indicate identity theft where a fraudster uses a known identity to open new accounts.
+Account Takeover: An account that is taken over will be accessed from a new device, i.e. a device it hasn’t been used with before. This can be treated as an additional risk compared to devices that have been regularly seen with said account. The lifespan during which a device can be recognized as ‘seen again’ is very relevant here, as shorter windows will make every account access look as if it came from a never-before-seen device.
+1st Party Fraud: A person committing 1st party fraud and then claiming to be a victim of identity theft can in some cases be detected by recognizing that their own device was used to commit said fraud.
+
+####  Recognize whether the same device is seen again in the context of multiple identities
```

Comment:
Hi Philippp, the original language here was submitted by another user. I have attempted to merge their comments in, so I don't have a stake in how this was originally worded, but at the same time I don't think I entirely agree with your point. 

If we are trying to detect abuse at account opening, or in the context of account take over, then 'inordinate' might be as small as 'two'. I think developers and organizations have a reasonable need to get a real count, as opposed to some sort of fuzzy threshold, so that they can build appropriate logical flows, or utilize the data points in models that consider a range of other factors. Deciding on a threshold for 'inordinate' seems to me to require some speculation on how a given capability is implemented, as opposed to just discussing the need in the abstract.

Reply:
Hi Philip, Sam,

Having the real count would be required since there is a relevant difference between 2 identifies sharing a device versus 100 identities sharing a device. While 2 users seems like a reasonable case of legitimate device sharing, when you get to more than 2 users for a device, legitimate and illegitimate cases of device sharing start blurring together. Deciding on an arbitrary threshold for "few" versus "many" users would not only lead to relevant information loss, but it would also empower fraudsters to play around this threshold. My stance is that fraud decisions need to combine the number of device users with other intelligence in order to make accurate decisions. 

I propose rephrasing my original statement to "Recognize how many identities share the same device"

Reply:
Could a bucketed count suffice?

Reply:
It could be workable if we start bucketing for numbers above a threshold of 10, but under that threshold we need the exact numbers.

Reply:
I'm ok leaving the wording as is. It's helpful to know that there is some flexibility in the entropy here, and more discussion (perhaps in a later issue/pull request) can illuminate the exact granularity. 

Sam, feel free to resolve this comment if nobody else has concerns with the wording as-is.